src/components/views/voip/IncomingCallBox.js

```diff
@@ -18,6 +19,7 @@ import PropTypes from 'prop-types';
 import MatrixClientPeg from '../../../MatrixClientPeg';
 import dis from '../../../dispatcher';
 import { _t } from '../../../languageHandler';
+import AccessibleButton from '../elements/AccessibleButton';
```

Comment:
Should this not be taken via `getComponent` ?

Reply:
mm, yes, it should. For some reason we've used import directly elsewhere in the codebase.